### DISCUSSION TITLE
Regenerating DNA

### DISCUSSION THREAD

**Discussion by VeriumETH** [nftchef/art-engine#116] [Help + Q&A] [answered]
Hey @nftchef 

Is it possible to regenerate the dna file if it has been deleted?

**Comment by nftchef** [comment, marked as answer]
Unfortunately no.
That's the key to recreating most things used in utils and is the output of the files/combinations used at runtime.
However, depending on what you need to do, you can approximate it and build utils around the metadata files.

**Reply by VeriumETH** [reply]
I understand, it was my mistake to delete the file without trying to understand the use first. Thank you!
